xbmc/video/dialogs/GUIDialogVideoInfo.cpp

```diff
@@ -597,7 +597,7 @@ void CGUIDialogVideoInfo::Play(bool resume)
     return;
   }
 
-  CFileItem movie(*m_movieItem->GetVideoInfoTag());
+  CFileItem movie(*m_movieItem);
```

Comment:
actually it's more like 10 years ago, sometimes prior to 2009 so it's all smashed into the svn to git conversion ;) but i looked into it. the problem is that we hop into the ShowResumeMenu, which calls into the Database to get resume points. these are stored by path, and not by the virtual db path, so i think that was the issue. but then again maybe the surrounding code has changed so the m_movieItem points to proper paths in the first place now?